app/src/main/java/com/github/se/cyrcle/ui/list/ListScreen.kt

```diff
@@ -39,7 +61,10 @@ fun SpotListScreen(
   val sortedFilteredParkingSpots =
       parkingSpots
           .map { parking ->
-            parking to TurfMeasurement.distance(referencePoint, parking.location.center)
+            // TODO: Use the user's location instead of a hardcoded reference point
```

Comment:
I would suggest to add this as an issue or has a draft issue so that we don't forget to do it